packages/client/src/object/Cache.test.ts

```diff
@@ -272,7 +272,7 @@ describe("AsyncCache", () => {
               "rejected",
             );
             expect(asyncCacheSpies.get.mock.settledResults[i].value)
-              .toMatchInlineSnapshot(`[Error: aError]`);
+              .toMatchObject(new Error("aError"));
```

Comment:
Oh did that break with new vitest?

Reply:
yeah, because you arent supposed to inline snapshot in a loop